src/analysis/retail/druid/restoration/modules/spells/Dreamstate.tsx

```diff
@@ -51,7 +59,7 @@ class Dreamstate extends Analyzer {
             </>
           }
         >
-          <>{formatNumber(this.owner.getPerMinute(this.totalCDR))} seconds</>
+          <>{formatNumber(this.owner.getPerMinute(this.totalCDR) / 1000)} seconds</>
```

Comment:
You don't need to divide by 1000. `getPerMinute` already expects input in milliseconds.

Reply:
Also would like to see a screenshot of how this looks

Reply:
for anyone else reading this - I am an idiot and `getPerMinute` actually takes 'unitless', not milliseconds. He does actually need to divide by 1000